tabulon_dxf/src/lib.rs

```diff
@@ -295,6 +295,20 @@ pub fn path_from_entity(e: &dxf::entities::Entity) -> Option<BezPath> {
 
             Some(bp)
         }
+        EntityType::Solid(ref s) => {
+            // FIXME: currently only support viewing from +Z.
+            if s.extrusion_direction.z != 1.0 {
+                return None;
+            }
+
+            let mut bp = BezPath::new();
+            bp.move_to(point_from_dxf_point(&s.first_corner));
+            bp.line_to(point_from_dxf_point(&s.third_corner));
+            bp.line_to(point_from_dxf_point(&s.fourth_corner));
+            bp.line_to(point_from_dxf_point(&s.second_corner));
```

Comment:
If the third and fourth corners are the same, then it is a triangle, I think:

https://help.autodesk.com/view/OARX/2018/ENU/?guid=GUID-E0C5F04E-D0C5-48F5-AC09-32733E8848F2

Reply:
Addressed now. We skip `fourth_corner` when it is equal to `third_corner`.